examples/client_channel/client_channel.js

```diff
@@ -13,10 +13,10 @@ const client = mc.createClient({
   password: process.argv[5]
 })
 
-client.registerChannel('MC|Brand', ['string', []])
-client.on('MC|Brand', console.log)
+client.registerChannel('minecraft:brand', ['string', []])
+client.on('minecraft:brand', console.log)
```

Comment:
A switch for versions with `MC|Brand` and `minecraft:brand` would be nice. There are still a lot of people how use mineflayer for 1.8 and 1.12.2. And those use `MC|Brand` instead of `minecraft:brand`

Reply:
Updated in https://github.com/PrismarineJS/node-minecraft-protocol/pull/1096/commits/dfd04ef4b979931c5dd7de8283e150bf92126573